configs/recognition/tsm/tsm_r50_dense_1x1x8_100e_kinetics400_rgb.py

```diff
@@ -18,7 +18,7 @@
         is_shift=True))
 # model training and testing settings
 train_cfg = None
-test_cfg = dict(average_clips=None)
+test_cfg = dict(average_clips='prob', test_crops=10)
```

Comment:
Since `DenseSampleFrames` is a strategy about samping, it not proper to directly set `test_crops=10`. I think it is better to add a `dense_sample` key for this case.

Reply:
Correct me if I miss something. 

`num_crops` is calculated by the following 4 params.
1. `twice_sample` in `SampleFrames`
2. `num_sample_positions` in `DenseSampleFrames`
3. `ThreeCrop/TenCrop/MultiGroupCrop` in `test_pipeline`
4. `num_clips` in `SampleFrames` or its subclass if `clip_len != 1`

So maybe in `test_cfg`, we should define 4 vars, corresponding to the above 4 params, which will make this more clearly for users.

Reply:
However, I think setting `test_crops` automatically may be a better solution